sql/importer: TestImportComputed failed

sql/importer.TestImportComputed [failed](https://tanzanite.cluster.engflow.com/invocations/default/1efe0400-f57a-4b78-a741-a4ab2808a59b?testReportRun=3&testReportShard=12&testReportAttempt=1#targets-Ly9wa2cvc3FsL2ltcG9ydGVyOmltcG9ydGVyX3Rlc3Q=) on master @ [feec8b9a9cf1fdb35b60bda3bba5733e6a9e434a](https://github.com/cockroachdb/cockroach/commits/feec8b9a9cf1fdb35b60bda3bba5733e6a9e434a):


```
=== RUN   TestImportComputed
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestImportComputed2015893916
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    import_stmt_test.go:4945: -- test log scope end --
test logs left over in: outputs.zip/logTestImportComputed2015893916
--- FAIL: TestImportComputed (580.46s)
=== RUN   TestImportComputed/pgdump
    import_stmt_test.go:4940: error executing query="IMPORT PGDUMP (\"http://127.0.0.1:42727\")" args=[]: pq: relation "users" already exists
    --- FAIL: TestImportComputed/pgdump (98.63s)
=== RUN   TestImportComputed/import-into-avro
    import_stmt_test.go:4940: error executing query="IMPORT INTO users (a, b) AVRO DATA (\"http://127.0.0.1:42727\")" args=[]: pq: job 1022746023144849409: could not mark as reverting: job 1022746023144849409: with status "running": expected session "010180ba1325f9307d4e7ebcd06e141a3d2e57" but found NULL
    panic.go:626: error executing query="DROP TABLE IF EXISTS users" args=[]: pq: relation "users" is offline: importing
    --- FAIL: TestImportComputed/import-into-avro (104.94s)
=== RUN   TestImportComputed/import-into-avro-expression-index
    import_stmt_test.go:4931: error executing query="CREATE TABLE users (a INT, b INT, INDEX ((a + b)))" args=[]: pq: relation "defaultdb.public.users" already exists
    panic.go:626: error executing query="DROP TABLE IF EXISTS users" args=[]: pq: relation "users" is offline: importing
    --- FAIL: TestImportComputed/import-into-avro-expression-index (6.23s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=12</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportComputed.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-44776

--- a/pkg/sql/importer/import_stmt_test.go
+++ b/pkg/sql/importer/import_stmt_test.go
@@ -4806,6 +4806,8 @@ func TestImportComputed(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	skip.UnderRace(t)
+
 	const nodes = 3
 
 	ctx := context.Background()

